snp_application/src/bt/snp_bt_ros_nodes.cpp

```diff
@@ -371,6 +409,10 @@ bool MotionPlanPubNode::setMessage(trajectory_msgs::msg::JointTrajectory& msg)
 bool FollowJointTrajectoryActionNode::setGoal(Goal& goal)
 {
   goal.trajectory = getBTInput<trajectory_msgs::msg::JointTrajectory>(this, TRAJECTORY_INPUT_PORT_KEY);
+  for (auto& point : goal.trajectory.points)
+  {
+    point.effort.clear();
+  }
```

Comment:
This was added because the ros2 controller in Humble complains if you try and send it trajectories. 
```
[ros2_control_node-11] [ERROR] [1716558874.750573816] [joint_trajectory_position_controller]: Trajectories with effort fields are currently not supported.
```
See source of this error [here](https://github.com/ros-controls/ros2_controllers/blob/731da8e43493f9da7e1be2eb8cf13154b1b240a8/joint_trajectory_controller/src/joint_trajectory_controller.cpp#L1474). Which doesn't exist in Foxy

Reply:
Yeah, this has been on the to-do list for a while, so I'm fine with merging this in too